ConfigSpace/configuration_space.py

```diff
@@ -614,9 +797,31 @@ def get_conditions(self) -> List[AbstractCondition]:
         return conditions
 
     def get_forbiddens(self) -> List[AbstractForbiddenComponent]:
+        """
+        Returns a list with all forbidden clauses from the configuration space.
+
+        Returns
+        -------
+        list[:ref:`Forbidden clauses`]
+            List with the forbidden clauses
+        """
         return self.forbidden_clauses
 
     def get_children_of(self, name: Union[str, Hyperparameter]) -> List[Hyperparameter]:
+        """
+        Returns a list with all children of a given hyperparameter.
+
+        Parameters
+        ----------
+        name : str
```

Comment:
Well, it could also be a hyperparameter. Please check the type annotations of the functions.